sensu/config.go

```diff
@@ -117,10 +121,14 @@ func (c *Config) Client() *client.Client {
 		return cfg.Client
 	}
 
+	// we need to add a default subscription, without one sensu server will crash
+	cn := os.Getenv("SENSU_CLIENT_NAME")
+	cs := split(os.Getenv("SENSU_CLIENT_SUBSCRIPTIONS"), ",")
+	cs = append(cs, "client:"+cn)
 	return &client.Client{
 		Name:          os.Getenv("SENSU_CLIENT_NAME"),
 		Address:       fetchEnv("SENSU_CLIENT_ADDRESS", "SENSU_ADDRESS"),
-		Subscriptions: split(os.Getenv("SENSU_CLIENT_SUBSCRIPTIONS"), ","),
+		Subscriptions: cs,
```

Comment:
you don't need to assign all these variables assignations you can easily build all the subscriptions in here:

```golang
Subscriptions: append(
    split(os.Getenv("SENSU_CLIENT_SUBSCRIPTIONS"), ","),
    fmt.Sprintf("client:%s", cfg.Client().Name),
)
```